stacker_blueprints/iam_roles.py

```diff
@@ -73,17 +44,21 @@ def generate_policy_statements(self):
 
         return []
 
-    def create_policy(self, name):
+    def create_policy(self, name=None):
         statements = self.generate_policy_statements()
         if not statements:
             return
 
         t = self.template
 
+        logical_name = "Policy"
+        if name:
+            logical_name += name
+
         policy = t.add_resource(
             iam.PolicyType(
-                "{}Policy".format(name),
-                PolicyName=Sub("${AWS::StackName}-${Name}-policy", Name=name),
+                logical_name,
+                PolicyName=Sub("${AWS::StackName}-policy"),
```

Comment:
This would result in a change as well - will this cause issues for existing users?

Reply:
Same as the above.